src/ntfs/read.rs

```diff
@@ -288,8 +288,8 @@ pub fn manage_comments(collections: &mut Collections, path: &path::Path) {
                     ObjectType::VehicleJourney => {
                         insert_comment_link(&mut collections.vehicle_journeys, comment_link)
                     }
-                    ObjectType::StopTime => warn!("comments are not added to StopTime yet"),
-                    ObjectType::LineGroup => warn!("line_groups.txt is not parsed yet"),
+                    ObjectType::StopTime => error!("comments are not added to StopTime yet"),
+                    ObjectType::LineGroup => error!("line_groups.txt is not parsed yet"),
```

Comment:
Sorry, these ones are warn because it should work but we warn that it's not supported yet.

Reply:
omg my bad >_<